src/data/padics/padic_norm.lean

```diff
@@ -297,6 +317,152 @@ theorem min_le_padic_val_rat_add {q r : ℚ}
 
 end padic_val_rat
 
+section padic_val_nat
+
+/--
+A rewrite lemma for `padic_val_nat p (q * r)` with conditions `q ≠ 0`, `r ≠ 0`.
+-/
+protected lemma mul (p : ℕ) [p_prime : fact p.prime] {q r : ℕ} (hq : q ≠ 0) (hr : r ≠ 0) :
```

Comment:
This name doesn't seem to follow the same convention as the one below it.

Reply:
It's named by analogy with the same function for `padic_val_rat` above.

Reply:
I think either everything should be of the form `padic_val_nat.mul` or `padic_val_nat_mul`. But you shouldn't mix them. I'm fine with changing the name below, or this one... your choice!